.gitignore

```diff
@@ -2,5 +2,7 @@
 *.egg-info
 .DS_Store
 .cache
+# ignore pycharm config files
```

Comment:
Good addition, let's add this on a separate PR to keep things clean.

Reply:
Here it is: https://github.com/Shopify/shopify_python/pull/111